File: FragmentNavigation/app/src/main/java/com/example/fragmentnavigation/AboutFragment.java
package com.example.fragmentnavigation;

import android.content.pm.PackageInfo;
import android.content.pm.PackageManager;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.example.fragmentnavigation.databinding.FragmentAboutBinding;

public class AboutFragment extends Fragment {

    FragmentAboutBinding binding;

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        binding = FragmentAboutBinding.inflate(inflater,container,false);
        return  binding.getRoot();
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        binding.textView4.setText(getActivity().getApplicationInfo().loadLabel(getActivity().getPackageManager()).toString());
        try {
            PackageInfo info = getActivity().getPackageManager().getPackageInfo(getActivity().getPackageName(),0);
            binding.textView5.setText(info.versionName);
            binding.textView6.setText(""+info.versionCode);
        } catch (PackageManager.NameNotFoundException e) {
            e.printStackTrace();
        }

        binding.button.setOnClickListener(v->{
            getActivity().getSupportFragmentManager().beginTransaction().replace(R.id.frag_container,new MenuFragment()).commit();
        });

    }

}

File: FragNavKotl/app/src/main/java/com/example/fragnavkotl/fragments/AboutFragment.kt
package com.example.fragmentnavigationkotlin.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.example.fragnavkotl.databinding.FragmentAboutBinding

class AboutFragment : Fragment(){
    lateinit var binding: FragmentAboutBinding
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        binding = FragmentAboutBinding.inflate(inflater,container,false)
        return binding.root
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        binding.applicationName.text = requireActivity().applicationInfo.loadLabel(requireActivity().packageManager)
        binding.versionName.text = requireActivity().packageManager.getPackageInfo(requireActivity().packageName,0).versionName
        binding.versionCode.text = requireActivity().packageManager.getPackageInfo(requireActivity().packageName,0).versionCode.toString()

    }



}